python/dlr/tflite_model.py

```diff
@@ -0,0 +1,133 @@
+import logging
+import os
+import tensorflow.lite as lite
+from .api import IDLRModel
+
+
+def _get_input_and_output_names(intrp):
+    """
+    Get input and output tensor names
+
+    Parameters
+    ----------
+    intrp : :py:class:`str`tf.lite.Interpreter`
+        TFLite Interpreter
+
+    Returns
+    -------
+    input_tensor_map : Input tensor Map which maps tensor_name to tensor_id
+    output_tensor_list : Output tensor list. List element is pair (name, id)
+    """
+    input_tensor_map = {}
+    output_tensor_list = []
+    for d in intrp.get_input_details():
+        input_tensor_map[d["name"]] = d["index"]
+    for d in intrp.get_output_details():
+        output_tensor_list.append((d["name"], d["index"]))
+    return input_tensor_map, output_tensor_list
+
+
+class TFLiteModelImpl(IDLRModel):
+    """
+    TFLiteModelImpl is a wrapper on top of Tensorflow Lite which implements IDLRModel API
+
+    Parameters
+    ----------
+    tflite_file : :py:class:`str`
+        Full path to TFLite file (.tflite file)
+    dev_type : :py:class:`str`
```

Comment:
It looks that `dev_type` and `dev_id` are TVM specific. We probably don't need to put them in the base class. This reduces confusion for users.

Reply:
Removed `dev_type` and `dev_id` from `TFLiteImpl.init`.
If user explicitly specifies `dev_type` or/and `dev_id` for `.tflite` file then the dispatcher will log Warning and ignore `dev_type` and `dev_id` parameters